File: datasets/IR-Plag/convertai-original-java-rest-typescript/case-03/plagiarized/L6/08/Kasus3L6.ts
import { Scanner } from 'readline-sync';

function hitungBMI(weight: number, height: number): number {
  const temp = height * 0.0254;
  return weight * 0.45359237 / (temp * temp);
}

function BMIKategori(BMI: number): void {
  console.log(`BMI is ${BMI}`);
  if (BMI > 30) {
    console.log('Obese');
  } else if (BMI > 25) {
    console.log('Overweight');
  } else if (BMI > 18.5) {
    console.log('Normal');
  } else {
    console.log('Underweight');
  }
}

function main(): void {
  const inp = new Scanner();
  console.log('Enter weight in pounds: ');
  const berat = inp.nextDouble();

  console.log('Enter feet: ');
  const kaki = inp.nextDouble();
  console.log('Enter inches: ');
  const inch = inp.nextDouble();

  // Print Hasil perhitungan
  BMIKategori(hitungBMI(berat, kaki * 12 + inch));
}

main();

File: datasets/IR-Plag/intellij-original-java-rest-kotlin/case-03/plagiarized/L6/08/Kasus3L6.kt
package IR

import java.util.*

/*
* To change this license header, choose License Headers in Project Properties.
* To change this template file, choose Tools | Templates
* and open the template in the editor.
*/

/**
 *
 * @author B15130F5DDB6B5F1622EF91DAC4C1AAE
 */
object Kasus3L6 {
    @JvmStatic
    fun main(args: Array<String>) {
        print("Enter weight in pounds: ")
        val inp = Scanner(System.`in`)
        val berat = inp.nextDouble()

        print("Enter feet: ")
        val kaki = inp.nextDouble()
        print("Enter inches: ")
        val inch = inp.nextDouble()


        //Print Hasil perhitungan
        BMIKategori(hitungBMI(berat, kaki * 12 + inch))
    }

    private fun hitungBMI(weight: Double, height: Double): Double {
        val temp = height * 0.0254
        return weight * 0.45359237 / (temp * temp)
    }

    private fun BMIKategori(BMI: Double) {
        println("BMI is $BMI")
        if (BMI > 30) {
            println("Obese")
        } else if (BMI > 25) {
            println("Overweight")
        } else if (BMI > 18.5) {
            println("Normal")
        } else {
            println("Underweight")
        }
    }
}
